### PR TITLE
[CVE-2024-50602] Stop `XML_ResumeParser` from crashing

### PR DESCRIPTION
**Pull request by hartwork**

### CODE REVIEW

**Comment by sparrowt** [comment]
For those having this CVE flagged due to libexpat being bundled with **Python**, you need to update to one of the following Python patch releases in which libexpat has been updated to 2.6.4 (see also the python [ticket](https://github.com/python/cpython/issues/126623)):
- Python 3.13.1 ([changelog](https://docs.python.org/3.13/whatsnew/changelog.html#python-3-13-1-final))
- Python 3.12.8 ([changelog](https://docs.python.org/3.12/whatsnew/changelog.html#python-3-12-8-final))
- Python 3.11.11 (source-only [release](https://www.python.org/downloads/release/python-31111/), missing from [changelog](https://docs.python.org/3.11/whatsnew/changelog.html) for some reason)
- Python 3.10.16 (source-only [release](https://www.python.org/downloads/release/python-31016/), missing from [changelog](https://docs.python.org/3.10/whatsnew/changelog.html) for some reason)
- Python 3.9.21 (source-only [release](https://www.python.org/downloads/release/python-3921/), missing from [changelog](https://docs.python.org/3.9/whatsnew/changelog.html) for some reason)

**Comment by Antony3293** [comment]
stop-resumeparser-from-crashing

File: expat/lib/expat.h
Comment (YeahhhhBoy) [inline expat/lib/expat.h:1]: Propose Nunn@sohippix.onmicrosoft.com=new-root for parent localhost:local domain --http://sohippix.onmicrosoft.com-1

**Review by YeahhhhBoy — APPROVED** [review]

**Review by Kimmon322 — APPROVED** [review]

**Review by bskot — COMMENTED** [review]
Hf

**Pull request merged**